Bottom: 49500b57f5e0b9578fca4e0b249a96f4d28336e9
Top:    ad39a3f5a75c8337a6df0f75c6fa4c84d8d6da5b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 02:03:29 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
